crates/libs/starknet/types/src/assets.rs

```diff
@@ -30,25 +33,34 @@ impl Asset {
         match self {
             Asset::Strk => "strk",
             Asset::Eth => "eth",
+            Asset::Btc => "btc",
+            Asset::Usdc => "usdc",
+            Asset::Usdt => "usdt",
         }
     }
 
     pub fn precision(&self) -> u8 {
         match self {
             Asset::Strk | Asset::Eth => 18,
+            Asset::Btc => 8,
+            Asset::Usdc | Asset::Usdt => 6,
```

Comment:
Can you explain those values?
Are you sure that the deployed er20 smart contracts for btc, usdc and usdt on starknet don't have a regular 18 digit precision?
Where did you get those values from?

Reply:
rename the variant of the enum too